### PR TITLE
Dont use default_catalog

### PR DESCRIPTION
**Pull request by spopezen**

### CODE REVIEW

File: Products/ZenModel/browser/Autocompleters.py [hunk 1]
```diff
@@ -34,12 +34,11 @@ def __call__(self, query='', dataRoot='devices'):
...
-        query = MatchGlob('titleOrId', query.rstrip('*') + '*')
+        catalog = IModelCatalogTool(self.context.dmd).devices
+        query = MatchGlob('name', query.rstrip('*') + '*')
         if isinstance(self.context, DeviceOrganizer):
             query = query & Eq('path', "/".join(self.context.getPhysicalPath()))
-        brains = catalog.evalAdvancedQuery(query)
+        brains = catalog.search(query=query)
 
         # TODO: Add titleOrId to the catalog's metadata.
         return  sorted((b.getObject().titleOrId() for b in brains),
```
Comment (johnlhamilton) [inline Products/ZenModel/browser/Autocompleters.py]: You could just do b.name if you add "fields='name'" to the search above

Comment (spopezen) [inline Products/ZenModel/browser/Autocompleters.py]: True dat

Comment (fraibacas) [inline Products/ZenModel/browser/Autocompleters.py]: IModelCatalogTool(self.context.dmd).devices should automatically add name

Comment (fraibacas) [inline Products/ZenModel/browser/Autocompleters.py]: https://github.com/zenoss/zenoss-prodbin/blob/model_catalog/Products/Zuul/catalog/model_catalog_tool.py#L61

File: Products/ZenModel/browser/Autocompleters.py [hunk 2]
```diff
@@ -34,12 +34,11 @@ def __call__(self, query='', dataRoot='devices'):
         if dataRoot != 'devices':
             import exceptions
             raise exceptions.ValueError("dataRoot should only be 'devices'")
-        catalog = getToolByName(self.context.dmd.Devices,
-            self.context.dmd.Devices.default_catalog)
-        query = MatchGlob('titleOrId', query.rstrip('*') + '*')
+        catalog = IModelCatalogTool(self.context.dmd).devices
```
Comment (johnlhamilton) [inline Products/ZenModel/browser/Autocompleters.py]: how about:

if isinstance(self.context, DeviceOrganizer):
    catalog = IModelCatalogTool(self.context).devices
else:
    catalog = IModelCatalogTool(self.context.dmd.Devices).devices

here, and then get rid of lines 39-40 below

**Pull request merged**